core/src/main/java/hudson/model/Job.java

```diff
@@ -53,19 +53,7 @@
 import hudson.search.SearchItems;
 import hudson.security.ACL;
 import hudson.tasks.LogRotator;
-import hudson.util.AlternativeUiTextProvider;
-import hudson.util.ChartUtil;
-import hudson.util.ColorPalette;
-import hudson.util.CopyOnWriteList;
-import hudson.util.DataSetBuilder;
-import hudson.util.DescribableList;
-import hudson.util.FormApply;
-import hudson.util.Graph;
-import hudson.util.ProcessTree;
-import hudson.util.RunList;
-import hudson.util.ShiftedCategoryAxis;
-import hudson.util.StackedAreaRenderer2;
-import hudson.util.TextFile;
+import hudson.util.*;
```

Comment:
Revert, we're not doing star imports in this repo.

Reply:
Reverted